### ISSUE TITLE
currentLocation always goes timeout 

### ISSUE THREAD

**Issue by mrfarukturgut** [malcommac/SwiftLocation#27]
Hi,
It was working fine in the simulator, but now it always goes timeout and returns error. Either in device or simulator. (I am not sure if that is relevant but there is no sim card in device)

here is the code: 

```
try! SwiftLocation.shared.currentLocation(.Room, timeout: 60.0, onSuccess: { (location) -> Void in
                postRequest(.UPDATE_LOCATION, additionalParameters: ["\(location?.coordinate.latitude)","\(location?.coordinate.longitude)"], onSuccess: { (json) -> () in
                        print(">>>location_successfully_updated")
                    }, onFailure: { (errorCode) -> () in
                    print(">>>couldn't_update_location")
                })
                }, onFail: { (error) -> Void in
                    print(">>>couldn't_get_location")
                    print(error?.localizedDescription)
            })
```

and the output:

`Optional(Error Domain=NSCocoaErrorDomain Code=0 "Timeout reached" UserInfo={NSLocalizedDescription=Timeout reached})`

**Comment by marvisais** [comment]
Hi, got any solution, i have the same issues :(

**Comment by malcommac** [comment]
I'm working on a new revamped version. Be patient,it's almost complete :)

**Comment by malcommac** [comment]
Fixed in 0.9.0, thank you
